### ISSUE TITLE
Crash `NoSuchMethodError: com.cleanroommc.modularui.value.BoolValue$Dynamic.<init>`

### ISSUE THREAD

**Issue by Krutoy242** [CleanroomMC/InventoryBogoSorter#52]
- `bogosorter-1.4.0`
- `modularui-2.2.1`

When pressing [...] button game crashing.
Tested on Draconic Evolution chest.

![image](https://github.com/CleanroomMC/InventoryBogoSorter/assets/936618/582428d4-67c0-4b35-93ba-d40713181b39)

```less
java.lang.NoSuchMethodError: com.cleanroommc.modularui.value.BoolValue$Dynamic.<init>(Ljava/util/function/BooleanSupplier;Ljava/util/function/Consumer;)V
    at com.cleanroommc.bogosorter.common.config.ConfigGui.createGeneralConfigUI(ConfigGui.java:96)
    at com.cleanroommc.bogosorter.common.config.ConfigGui.buildUI(ConfigGui.java:70)
    at com.cleanroommc.modularui.screen.ModularScreen.<init>(ModularScreen.java:109)
    at com.cleanroommc.bogosorter.common.config.ConfigGui.<init>(ConfigGui.java:47)
    at com.cleanroommc.bogosorter.common.sort.ButtonHandler.onActionPerformed(ButtonHandler.java:95)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_2907_ButtonHandler_onActionPerformed_Pre.invoke(.dynamic)
```

[debug.log](https://github.com/CleanroomMC/InventoryBogoSorter/files/12782989/debug.log)

**Comment by brachy84** [comment]
f

**Comment by brachy84** [comment]
Should be fixed with mui 2.2.2 and bogo 1.4.1. Sorry for so many issues
